addon/components/bs4/bs-form/element/help-text.js

```diff
@@ -0,0 +1,5 @@
+import FormElementHelpText from 'ember-bootstrap/components/base/bs-form/element/help-text';
+
+export default FormElementHelpText.extend({
+  classNames: ['form-text']
```

Comment:
It seems it was part of the `errors` component in BS4, but removed later. See this: https://github.com/kaliber5/ember-bootstrap/commits/master/addon/components/bs4/bs-form/element/errors.js